uitools/toolkitwidgets/docs/FloorFilter.md

```diff
@@ -0,0 +1,11 @@
+[![ArcGIS Maps SDK for Qt](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt-0b5394)](https://developers.arcgis.com/qt/) [![ArcGIS Maps SDK for Qt toolkit](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt%20toolkit-ea4d13)](https://github.com/Esri/arcgis-maps-sdk-toolkit-qt) [![Qt Widgets UI components](https://img.shields.io/badge/Qt%20Qt%20Widgets%20UI%20components-ea4d13)](../../toolkitwidgets/)
+
+# FloorFilter - Qt Widgets UI control/Class
+
+The [FloorFilter - Qt Widgets UI control/Class](https://developers.arcgis.com/qt/toolkit/api-reference/esri-arcgisruntime-toolkit-floorfilter.html) tool allows for the browsing workflow where sites and facilities are manually selected, automatic facility selection based on the GeoView's current viewpoint. This mode is useful when you are exploring the GeoView but do not know the name of the site or facility. For the same reason, it is possible to list all the facilities available in the `FloorManager` using the button on the siteView.
```

Comment:
Please be sure to include any updates to the Quick version of FloorFilter to Widgets as well

Reply:
Fixed with commit: https://github.com/Esri/arcgis-maps-sdk-toolkit-qt/pull/660/commits/ff1b75977c0a65f68ceb1700d99499483be324f0